# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/moz.build
# Commit: c414d82a5325
# Full Hash: c414d82a532508523d40ce63111cb30b2a1076be
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-02-22 05:26:22
# Regressor Bug: 1521879
# File Overlap Count: 1
# Description:
#   Bug 1521879 - Part 1: IPC-based MediaTransport implementation r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D17273
# ==============================================================================

diff -r 3a182b05e2c1 -r c414d82a5325 dom/media/moz.build
--- a/dom/media/moz.build	Thu Feb 21 19:01:32 2019 +0200
+++ b/dom/media/moz.build	Thu Feb 21 16:42:12 2019 +0000
@@ -175,6 +175,7 @@
 
 EXPORTS.mozilla.media.webrtc += [
     'webrtc/WebrtcGlobal.h',
+    'webrtc/WebrtcIPCTraits.h',
 ]
 
 if not CONFIG['MOZ_WEBRTC']:
@@ -183,7 +184,7 @@
   ]
 
 IPDL_SOURCES += [
-    'webrtc/PWebrtcGlobal.ipdl'
+    'webrtc/PWebrtcGlobal.ipdl',
 ]
 
 EXPORTS.mozilla.dom += [